.github/workflows/ci.yml

```diff
@@ -67,6 +67,8 @@ jobs:
             value: ubuntu1604-64vpnserver.ma{hostname=vpnserver}-ubuntu1604-64vpnclienta.a{hostname=vpnclienta}
           - name: Ubuntu 18.04
             value: ubuntu1804-64vpnserver.ma{hostname=vpnserver}-ubuntu1804-64vpnclienta.a{hostname=vpnclienta}
+          - name: Ubuntu 20.04
+            value: ubuntu2004-64vpnserver.ma{hostname=vpnserver}-ubuntu2004-64vpnclienta.a{hostname=vpnclienta}
```

Comment:
FYI, modulesync now has some code that generates the build matrix automatically.

For now, this will not solve the expected failure we are seeing here, but it is being worked on :grin: